File: app/src/main/java/com/example/xuyangtakeout/model/bean/Order.java
package com.example.xuyangtakeout.model.bean;

import java.util.List;

public class Order {
	public String id;
	public String type;
	public Seller seller;
	public Rider rider;
	public List<GoodsInfo> goodsInfos;
	public Distribution distribution;
	public OrderDetail detail;

	public class Rider {
		public int id;
		public String name;
		public String phone;
		// public Location location;
	}

	public class OrderDetail {
		public String username;
		public String phone;
		public String address;
		public String pay;
		public String time;
	}

	public class Distribution {
		// 配送方式
		public String type;
		// 配送说明
		public String des;
	}

}

File: app/src/main/java/com/example/xuyangtakeout/model/bean/Order.kt
package com.example.xuyangtakeout.model.bean

class Order {
    var id: String? = null
    var type: String? = null
    var seller: Seller? = null
    var rider: Rider? = null
    var goodsInfos: List<GoodsInfo>? = null
    var distribution: Distribution? = null
    var detail: OrderDetail? = null

    inner class Rider {
        var id: Int = 0
        var name: String? = null
        var phone: String? = null
        // public Location location;
    }

    inner class OrderDetail {
        var username: String? = null
        var phone: String? = null
        var address: String? = null
        var pay: String? = null
        var time: String? = null
    }

    inner class Distribution {
        // 配送方式
        var type: String? = null
        // 配送说明
        var des: String? = null
    }

}
